src/frontend/pages/game.tsx

```diff
@@ -0,0 +1,205 @@
+import Head from 'next/head';
+import styles from '../styles/Game.module.css';
+import Slider from 'react-input-slider';
+import Modal from 'react-modal';
+import {
+  buildStyles,
+  CircularProgressbarWithChildren,
+} from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import React, { ReactElement, useEffect, useRef, useState } from 'react';
+import Image from 'next/image';
+
+const MAX_TAKE_VAL = 10;
+const INIT_TIME_LEFT = 10;
+
+export default function Home(): ReactElement {
+  const [takeVal, setTakeVal] = useState(MAX_TAKE_VAL);
+  const [timeLeft, setTimeLeft] = useState(INIT_TIME_LEFT);
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+
+  useEffect(() => {
+    const interval = setInterval(
+      () => setTimeLeft((timeLeft) => timeLeft - 1),
+      1000,
+    );
+
+    return () => clearInterval(interval);
+  }, [timeLeft]);
+
+  if (timeLeft === 0) {
+    setTakeVal(Math.floor(Math.random() * 11));
+    setTimeLeft(10);
+  }
+
+  const inputOnChange = (eventVal: string) => {
+    const intVal = parseInt(eventVal);
+
+    // TODO: You can't type a '-', not sure we need this first condition
+    if (intVal < 0) {
+      alert('Input cannot be negative');
+      setTakeVal(0);
+    } else if (intVal > 10) {
+      alert('Input cannot be greater than 10');
```

Comment:
Also if we need the input, there has to be a better way to do this than an alert (though if this is a temporary measure with intent to make nicer then you can ignore)

Reply:
Yeah, I think we eventually want to incorporate a custom message (maybe as a modal)